client/wsclient_test.go

```diff
@@ -17,6 +17,19 @@ import (
 	"github.com/open-telemetry/opamp-go/protobufs"
 )
 
+type TestLogger struct {
```

Comment:
```suggestion
type testLogger struct {
```
We don't want to export this, it is just for local usage.

Reply:
Fixed it. Made the change